src/test/scala/uk/gov/hmrc/test/ui/pages/section1/DetailKeys.scala

```diff
@@ -51,6 +51,6 @@ object DetailKeys {
   // /enter-a-mucr
   val Mucr: DetailKey = DetailKey("MUCR", section1)
 
-  // /enterry-into-declarants-records
+  // /entery-into-declarants-records
```

Comment:
`/entry`

Reply:
Done

Reply:
But isn't `/entery` yet? Should not be `/entry`?